manifests/repo.pp

```diff
@@ -18,23 +18,15 @@
     descr    => "Pulpcore ${version}",
     baseurl  => pick($baseurl, "https://yum.theforeman.org/pulpcore/${version}/${dist_tag}/\$basearch"),
     enabled  => '1',
-    gpgcheck => if $gpgkey == '' { '0' } else { '1' },
-    gpgkey   => pick($gpgkey, "https://yum.theforeman.org/pulpcore/${version}/GPG-RPM-KEY-pulpcore"),
+    gpgcheck => if $gpgkey == '' or version == 'nightly' { '0' } else { '1' },
+    gpgkey   => if version == 'nightly' { 
```

Comment:
@ekohl what do you think about this method of setting the gpgkey for nightly? I've been doing some Puppet trial and error.

Reply:
It needs to be `$version` to pick up the variable ;)

Reply:
ah thank you!

Reply:
I am amazed it didn't yell at you that it has no friggin idea what `version` is.